### ISSUE TITLE
Flow doesn't do anything

### ISSUE THREAD

**Issue by huachuman** [Make-md/makemd#210]
I enabled the flow editor but all it does is put two little buttons at the top right of the block. Even if I click flow editor, nothing happens. It just stays a regular block embed. I click anywhere on the block and it just does the default behavior, which is selecting the ![[title]] of the embed. I've tried restarting Obsidian to no effect.

**Comment by javinavasviolin** [comment]
Yes, I am also having problems with the flow editor, it even deletes characters from the title when I click the button.

**Comment by huachuman** [comment]
I thought that make.md would get better with development but it seems to be going backwards.

**Comment by jp-cen** [comment]
Thanks for reporting this. It seems like Obsidian's markdown renderer was updated and caused the position calculation to be incorrect. This has been fixed in the new patch, although there is still one case where this still happens and I'll look into it for the next patch.

**Comment by javinavasviolin** [comment]
Thank you so much. When I click on "toggle flow" I never get the same results. Usually it doesn't allow me to edit the block and deletes characters previously. In the video you can see that when I click on "toggle flow" it deletes characters on the heading.


https://github.com/Make-md/makemd/assets/145867557/77dca4df-7fc4-4457-b7ee-e88e5cf17183

**Comment by jp-cen** [comment]
This is post patch? This is the issue I’m looking into, it seems like obsidian no longer deterministically returns the position of the current embed. You can always add the ! In front of the embed block. I’m looking for a new strategy to patch this

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jp-cen** [comment]
@javinavasviolin the version you're using seems to be before the patch, let me know if you get a chance to update and test the fix

**Comment by javinavasviolin** [comment]
I updated, it is the last version

**Comment by javinavasviolin** [comment]
Hi @jp-cen , I just updated and it still is behaving weird, now it adds "!" each time that I click it and doesn't allow me to edit the block inside.

https://github.com/Make-md/makemd/assets/145867557/6391e62f-efc0-47a3-ae72-4aa690e2b683

**Comment by jp-cen** [comment]
The ! should be inserted infront of the embed block but the underlying editor now returns two possible positions instead of reliably returning one. I’ll be rewriting the block position calculation today to address this. The most reliable way to create an editable block is to use the slash menu and directly create one from scratch.

**Comment by javinavasviolin** [comment]
I tried from slash, now doesn't appear the option of "flow note" only "lexisting note" that I guess is the same.

-Now the first time that I create it allows me to edit, but when I click again it I cannot and as you can see the "!" changes the color and becomes dark. 
- When I delete the "!" it allows to edit again.
- When I delete the linked note you can see a bunch of "!" that weren't there before as if they would be kind of residual.

I am not a programmer but I love Make.md since the first moment I saw it and I believe in your project, you are amazing guys!

https://github.com/Make-md/makemd/assets/145867557/1797a90d-5e8d-4673-8761-9be5cba3d63d

**Comment by jp-cen** [comment]
Flow editor, make menu and inline styler are now available as a standalone plug-in at https://github.com/Make-md/Obsidian-Basics

This will allow easier contribution, debugging and improvements to these features and also overall stability of the main make.md plug-in.

It is available now as a beta and will be submitted to Obsidian community plug-ins shortly. Feel free to provide any feedback or suggestions in the new repo.
